superset/assets/javascripts/dashboard/reducers/initState.js

```diff
@@ -0,0 +1,106 @@
+import { chart } from '../../chart/chartReducer';
+import { initSliceEntities } from './sliceEntities';
+import { getParam } from '../../modules/utils';
+import { applyDefaultFormData } from '../../explore/stores/store';
+import { getColorFromScheme } from '../../modules/colors';
+import layoutConverter from '../util/dashboardLayoutConverter';
+
+export default function(bootstrapData) {
```

Comment:
I think you'll get some .eslint errors for this soon, but debugging is easier if you name your functions

Reply:
fixed lint.